src/libdrakvuf/drakvuf.c

```diff
@@ -585,7 +585,7 @@ int drakvuf_get_address_width(drakvuf_t drakvuf)
 bool drakvuf_get_current_process_data(drakvuf_t drakvuf, drakvuf_trap_info_t* info, proc_data_priv_t* proc_data)
 {
     addr_t process_base = drakvuf_get_current_process(drakvuf, info);
-    return drakvuf_get_process_data_priv(drakvuf, process_base, proc_data);
+    return drakvuf_get_process_data_priv(drakvuf, process_base, proc_data) && drakvuf_get_current_thread_id(drakvuf, info, (uint32_t*)&proc_data->tid);
```

Comment:
`drakvuf_get_process_data_priv` itself should gather the `tid`, there shouldn't be a separate call to `drakvuf_get_current_thread_id` here.

Reply:
In windows, we need "drakvuf_trap_info_t* info" to find the thread it. Right now drakvuf_get_process_data_priv uses process base to find all data.

Reply:
That's fine, you can still move the function even if there is no Windows version of it yet, just put a TODO comment that Windows version is needed

Reply:
Since I have added Tid support for windows. I don't think I can remove.

Reply:
I will put a TODO comment and I will try to solve this soon myself.